core/src/test/java/io/questdb/griffin/engine/functions/geohash/GeoHashNativeTest.java

```diff
@@ -39,35 +39,28 @@
     static final double lat = 31.23;
     static final double lon = 121.473;
     static final StringSink sink = new StringSink();
+    private static final StringConverter toString = GeoHashes::appendChars;
+    private static final StringConverter toBitString = GeoHashes::appendBinary;
 
-    // TODO: technical debt, everything but testSlideFoundBlocks, testAiota
```

Comment:
Note, every test in here but testSlideFoundBlocks and testAiota needs to go to GeoHashesTest, and then some redundancy needs to go out of there.